src/shared/actions/transfers.js

```diff
@@ -665,7 +665,7 @@ export const makeTransaction = (seedStore, receiveAddress, value, message, accou
             })
             .then(({ trytes, transactionObjects }) => {
                 cached.trytes = trytes;
-                cached.transactionObjects = transactionObjects;
+                cached.transactionObjects = transactionObjects.slice().reverse();
```

Comment:
Are we reversing the transaction objects to match the order of trytes? At which point is the order changed?

Reply:
Reversing for correct storage order